src/main/java/com/ssbb/kanban/dao/impl/ProjectDAO.java

```diff
@@ -8,14 +8,20 @@
 
 public class ProjectDAO extends DAOImpl<Storable> {
 
-	private final static Class<Project> projectClass = Project.class;
-	
-	public Project getProjectByName(String name) {
-		Query q = em.createNativeQuery(
-				"SELECT * FROM PROJECTS WHERE NAME = :name", projectClass);
-		q.setParameter("name", name);
-		
-		return (Project) q.getSingleResult();
-	}
+    private final static Class<Project> projectClass = Project.class;

```

Comment:
I think the javadocs and the auto format may have moved the lines number about. I took the latest update from develop on this